api/extension-guides/chat.md

```diff
@@ -269,6 +269,77 @@ cat.followupProvider = {
+
+To implement participant detection, you specify the `disambiguation` property in the extension `package.json` file. The `disambiguation` property contains a list of detection categories, each with a description and examples.
+
+| Property | Description | Examples |
+|----------|-------------|----------|
+| `category` | The detection category. If the participant serves different purposes, you can have a category for each.  | <ul><li>`cat`</li><li>`workspace_questions`</li><li>`web_questions`</li></ul> |
+| `description` | A detailed description of the kinds of questions that are suitable for this participant. | <ul><li>`The user wants to learn a specific computer science topic in an informal way.`</li><li>`The user just wants to relax and see the cat play.`</li></ul> |
+| `examples` | A list of representative example questions. | <ul><li>`Teach me C++ pointers using metaphors`</li><li>`Explain to me what is a linked list in a simple way`</li><li>`Can you show me a cat playing with a laser pointer?`</li></ul> |
+
+You can define participant detection for the overall chat participant, for specific commands, or a combination of both.
+
+The following code snippet shows how to implement participant detection at the participant level.
+
+```json
+"contributes": {
+    "chatParticipants": [
+        "id": "chat-sample.cat",
+        "fullName": "Cat",
+        "name": "cat",
+        "description": "Meow! What can I teach you?",
+
+        "disambiguation": [
+            {
+                "category": "cat",
+                "description": "The user wants to learn a specific computer science topic in an informal way.",
+                "examples": [
+                    "Teach me C++ pointers using metaphors",
+                    "Explain to me what is a linked list in a simple way",
+                    "Can you explain to me what is a function in programming?"
+                ]
+            }
+        ]
+    ]
+}
+```
+
+This code snippet shows how to implement participant detection at the command level.
+
+```json
+"contributes": {
+    "chatParticipants": [
+        "id": "chat-sample.cat",
+        "fullName": "Cat",
+        "name": "cat",
+        "description": "Meow! What can I teach you?",
+
+        "commands": [
+            {
+                "name": "play",
+                "description": "Do whatever you want, you are a cat after all",
+                "disambiguation": [
+                    {
+                        "category": "cat_play",
+                        "description": "The user just wants to relax and see the cat play.",
+                        "examples": [
+                            "Enough learning, let the cat play with a ball of yarn",
+                            "Can you show me a cat playing with a laser pointer?"
+                        ]
+                    }
+                ]
+            }
+        ],
+    ]
+}
+```
+
```

Comment:
Should we have a sentence on suggestions on how to write good disambiguation content. And to urge the author to test it out and optimise the disambiguation based on how it behaves in VS Code?

Reply:
+1

Reply:
I added some general guidelines. @isidorn can you verify?